### PR TITLE
typo

### PR DESCRIPTION
**Pull request by 1234567-glitch**
s0md3v i am not sure if i have permission to do some edit on to backslash?

### CODE REVIEW

**Comment by 1234567-glitch** [comment]
I Found typo

**Review by thehappydinoa — CHANGES_REQUESTED** [review]
You should remove the typo message from the README

**Comment by 1234567-glitch** [comment]
i will work on this issue, thanks

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
+found a typo issue file
+s0md3v/AwesomeXSS
 # AwesomeXSS
 This repository is a collection of Awesome XSS resources. Contributions are welcome and should be submitted via an issue.
 
 ### Donations
-You can encourage me to create more such amazing programs buy donating a few bucks.
+You can encourage me to create more such amazing programs by donating a few bucks.
```
Comment (1234567-glitch) [inline README.md:7]: changed buy to by

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
+found a typo issue file
```
Comment (thehappydinoa) [inline README.md]: This should be in the pull request message and should be removed from the README

**Review by thehappydinoa — CHANGES_REQUESTED** [review]

**Comment by 1234567-glitch** [comment]
> You should remove the typo message from the README

thanks,  i am still learning though. yes i can work on it

File: README.md (hunk 2 above)
Comment (1234567-glitch) [reply README.md]: ok thanks thehappydinoa will do like wise

**Pull request closed without merging**